package/src/specs/RNCandle.nitro.ts

```diff
@@ -6,6 +6,225 @@ export type AppUser = {
+};
+
+type AssetAccountDetails = {
+  fiatAccountDetails?: FiatAccountDetails;
+  marketAccountDetails?: MarketAccountDetails;
+};
+
+export type LegalAccountKind =
+  | "individual"
+  | "joint"
+  | "traditionalIra"
+  | "rothIra";
+
+export type AssetAccount = {
+  legalAccountKind: LegalAccountKind;
+  nickname: string;
+  details: AssetAccountDetails;
+};
+
+type AssetAccountKind = "fiat" | "stock" | "crypto";
+
+export type AssetAccountQuery = {
+  linkedAccountIDs?: string;
+  assetKind?: AssetAccountKind;
+};
+
+export type FiatAsset = {
+  assetKind: string; // fiat
+  serviceTradeID?: string;
+  serviceAccountID: string;
+  currencyCode: string;
+  amount: number;
+  linkedAccountID: string;
+  service: Service;
+};
+
+export type MarketTradeAsset = {
+  assetKind: string; // "stock" | "crypto"
+  serviceAccountID: string;
+  serviceAssetID: string;
+  symbol: string;
+  amount: number;
+  serviceTradeID: string;
+  linkedAccountID: string;
+  name: string;
+  color: string;
+  logoURL: string;
+};
+
+type Coordinates = {
+  latitude: number;
+  longitude: number;
+};
+
+type Address = {
+  value: string;
+};
+
+export type TransportAsset = {
+  assetKind: string; // "transport"
+  serviceTradeID: string;
+  serviceAssetID: string;
+  name: string;
+  description: string;
+  imageURL: string;
+  originCoordinates: Coordinates;
+  originAddress: Address;
+  destinationCoordinates: Coordinates;
+  destinationAddress: Address;
+  seats: number;
+  linkedAccountID: string;
+};
+
+export type OtherAsset = {
+  assetKind: string; // "other"
+};
+
+export type NothingAsset = {
+  assetKind: string; // "nothing"
+};
+
+export type TradeAsset = {
+  fiatAsset?: FiatAsset;
+  marketTradeAsset?: MarketTradeAsset;
+  transportAsset?: TransportAsset;
+  otherAsset?: OtherAsset;
+  nothingAsset?: NothingAsset;
+};
+
+type MerchantLocation = {
+  countryCode?: string;
+  countrySubdivisionCode?: string;
+  localityName?: string;
+};
+
+export type MerchantCounterparty = {
+  kind: string; // "merchant";
+  name: string;
+  logoURL: string;
+  location?: MerchantLocation;
+};
+
+export type UserCounterparty = {
+  kind: string; // "user"
+  legalName: string;
+  avatarURL: string;
+  username: string;
+};
+
+export type ServiceCounterparty = {
+  kind: string; // "service"
+  service: Service;
+};
+
+export type Counterparty = {
+  merchantCounterparty?: MerchantCounterparty;
+  userCounterparty?: UserCounterparty;
+  serviceCounterparty?: ServiceCounterparty;
+};
+
+export type TradeState = "success" | "inProgress" | "failure";
+
+export type Trade = {
+  dateTime: string;
+  state: TradeState;
+  counterparty: Counterparty;
+  lost: TradeAsset;
+  gained: TradeAsset;
+};
+
+export type TradeQuery = {
+  linkedAccountIDs?: string;
+  dateTimeSpan?: string;
+  gainedAssetKind?: string; // "fiat" | "stock" | "crypto" | "transport" | "other" | "nothing"
+  lostAssetKind?: string; // "fiat" | "stock" | "crypto" | "transport" | "other" | "nothing"
```

Comment:
#req define enum here rather than in JS wrapper layer... you're defining other similar enums in this file so clearly Nitro can handle them

Reply:
I had that before, but couldn't get the codegen to compile.